codelists/models.py

```diff
@@ -472,10 +472,17 @@ def codes(self):
             return self._new_style_codes()
 
     def _old_style_codes(self):
-        if self.coding_system_id in ["ctv3", "icd10", "snomedct"]:
+        if self.coding_system_id in ["bnf", "ctv3", "icd10", "snomedct"]:
```

Comment:
I don't know if there was a reason that `bnf` is missing here, but it is a coding system with a hierarchy, so old-style uploads for bnf codelists will try to build a hierarchy, but have no codes to build from,  and will [error](https://sentry.io/organizations/ebm-datalab/issues/3777833872/?project=5248013)